core/thread.py

```diff
@@ -474,67 +530,59 @@ def __getitem__(self, item):
             reason='Creating a thread channel'
         )
 
-        thread.channel = channel
+        thread = Thread(self, recipient, channel)
+        self.cache[recipient.id] = thread
 
-        log_url, log_data = await asyncio.gather(
-            self.bot.modmail_api.create_log_entry(recipient, channel, creator or recipient),
-            self.bot.modmail_api.get_user_logs(recipient.id)
-            # self.get_dominant_color(recipient.avatar_url)
-        )
+        log_url = await self.bot.api.create_log_entry(recipient, channel,
```

Comment:
not sure why you are no longer using `asyncio.gather`, you should try to utilise asyncio to do things concurrently for more speed.

Reply:
I got to disagree with this, `asyncio.gather` does not make things faster. The native way is both more clear and grant you more speed. Here's a simple test to demonstrate:

https://repl.it/repls/NavyblueElatedCollaborativesoftware

    0.16229477599972597  <- await
    0.5546770680002737   <- gather

Reply:
It does make things faster in fact, the test you linked is flawed since there is no network io involved. `asyncio.gather` calls the two coroutines simultaneously which makes it faster.

Reply:
I suppose you meant by such cases:

https://repl.it/repls/NavyblueElatedCollaborativesoftware

Then that's fair, I'll change the ones that utilize requests back since concurrency faster when running these tasks, and if there's anything with bare computing, those will get changed to awaits.